src/plugins/components/text.ts

```diff
@@ -0,0 +1,128 @@
+import plugin from 'tailwindcss/plugin'
+import { defu } from 'defu'
+import { type PluginOption, defaultPluginOptions } from '../options'
+
+const defaultTextConfig = {
+  textXS: 'xs',
+  textSM: 'sm',
+  textMD: 'base',
+  textLG: 'lg',
+  textXL: 'xl',
+  text2XL: '2xl',
+  text3XL: '3xl',
+  text4XL: '4xl',
+  text5XL: '5xl',
+  text6XL: '6xl',
+  text7XL: '7xl',
+  text8XL: '8xl',
+  text9XL: '9xl',
+  textLight: 'light',
+  textNormal: 'normal',
+  textMedium: 'medium',
+  textSemibold: 'semibold',
+  textBold: 'bold',
+  textExtrabold: 'extrabold',
+  textLeadNone: 'none',
+  textLeadNormal: 'normal',
+  textLeadTight: 'tight',
+  textLeadSnug: 'snug',
+  textLeadLoose: 'loose',
+}
+
+export default plugin.withOptions(
+  function (options: PluginOption) {
+    const { prefix } = defu(options, defaultPluginOptions)
+
+    return function ({ addComponents, theme }) {
+      const config = theme('shurikenUi.text') satisfies typeof defaultTextConfig
+
+      addComponents({
+        [`.${prefix}-text`]: {
+          [`@apply font-sans`]: {},
+
+          [`&.${prefix}-content-xs`]: {
+            [`@apply ${config.textXS}`]: {},
```

Comment:
classes prefix are missing on all the file

```suggestion
            [`@apply text-${config.textXS}`]: {},
```